src/pcapi/core/users/api.py

```diff
@@ -283,3 +289,65 @@ def user_expenses(user: User):
         )
 
     return limits
+
+
+def create_pro_user(body: ProUserCreationBodyModel) -> User:
+    objects_to_save = []
+
+    new_user = User(from_dict=body.dict(by_alias=True))
```

Comment:
Et on peut sûrement préciser que c'est un pro ici :)
```suggestion
    new_pro_user = User(from_dict=body.dict(by_alias=True))
```